docs/en/latest/concepts/annotations.md

```diff
@@ -360,3 +360,32 @@ spec:
             port:
               number: 50053
 ```
+
+## Cross namespace reference service
+
+The value of the annotation should be the namespace of the service.
```

Comment:
"This annotation can be used to route to services in a different namespace."

Or something that explains what this annotation does.